src/platforms/javascript/guides/react/configuration/integrations/react-router.mdx

```diff
@@ -34,6 +34,51 @@ Make sure `Sentry.reactRouterV6Instrumentation` is initialized by your `Sentry.i
 
 </Alert>
 
+### Usage with React Router 6.4 Data API
+_(Available in version 7.21.0 and above)_
+
+
+If you create your router instance with [`createBrowserRouter`](https://reactrouter.com/en/main/routers/create-browser-router) from `react-router-dom` package, you can use `Sentry.wrapCreateBrowserRouter` to wrap it with the instrumentation:
+
+```jsx
+import { createBrowserRouter } from 'react-router-dom';
+import * as Sentry from '@sentry/react';
+import { BrowserTracing } from '@sentry/tracing';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new BrowserTracing({
+      routingInstrumentation: Sentry.reactRouterV6Instrumentation(
+          React.useEffect,
+          useLocation,
+          useNavigationType,
+          createRoutesFromChildren,
+          matchRoutes,
+      ),
+    }),
+  ],
+  tracesSampleRate: 1.0,
+});
+
+
+const sentryCreateBrowserRouter = Sentry.wrapCreateBrowserRouter(
+  createBrowserRouter
+);
+
+const router = sentryCreateBrowserRouter([
+  // ...
+]);
+
+```
+
+<Alert level="warning" title="Note">
+
+[`createHashRouter`](https://reactrouter.com/en/main/routers/create-hash-router) is not yet supported by Sentry. On the other hand [`createMemoryRouter`](https://reactrouter.com/en/main/routers/create-memory-router) can be instrumented using `wrapCreateBrowserRouter` function.
```

Comment:
we should create a GH issue about `createHashRouter` that we can link folks to here.